hooks/wordpress/useWordpressReferences.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { WordpressService } from "../../services/WordpressService";
+import {useCallback, useEffect, useState} from "react";
+import { LogService } from "../../../core/LogService";
+import { WordpressReferenceDTO } from "../../../core/wordpress/dto/WordpressReferenceDTO";
+import { routeValidation } from "./routeValidation";
+
+const LOG = LogService.createLogger('useWordpressReferencesList');
+
+export function useWordpressReferencesList(url?:string, endpoint?:string, refresh?:boolean):
```

Comment:
I still think this should return the refresh callback like all of our other hooks, and this property refresh boolean should not be here.